apps/src/EditorAnnotator.js

```diff
@@ -830,10 +830,10 @@ export default class EditorAnnotator {
       return EditorAnnotator.code_;
     }
 
-    let code = EditorAnnotator.annotator().getCode();
+    let code = EditorAnnotator.annotator()?.getCode();
     EditorAnnotator.code_ = code;
 
-    if (options.stripComments) {
+    if (options.stripComments && code) {
```

Comment:
An unrelated but potential folly where things crash if the annotations attempt to happen before the code editor actually loads.